src/cdk/v2/destinations/dynamic_yield/procWorkflow.yaml

```diff
@@ -0,0 +1,188 @@
+              "currency": .message.properties.currency,
+            }
+          }
+
+      - name: updateOrderCompletedEventProperties
+        condition: $.outputs.prepareTrackPayload.getTrimmedEvent === 'order_completed'
+        template: |
+          {
+            ...$.outputs.prepareTrackPayload.populateEventProperties,
+            "properties": {
+              ...$.outputs.prepareTrackPayload.populateEventProperties.properties,
+              "uniqueTransactionId" : ^.message.properties.order_id,
+              "value": .message.properties.value || .message.properties.revenue || .message.properties.price
+            }
+          }
+
+      - name: updateOtherEventProperties
+        condition: $.outputs.prepareTrackPayload.getTrimmedEvent !== 'order_completed'
+        template: |
+          {
+          ...$.outputs.prepareTrackPayload.populateEventProperties,
+          "properties": {
+            ...$.outputs.prepareTrackPayload.populateEventProperties.properties,
+            "quantity" : .message.properties.quantity,
+            "productId": .message.properties.sku || .message.properties.product_id
+            }
+          }
+
+      - name: populateCartObj
+        condition: Array.isArray(.message.properties.products)
+        template: |
+          const eventProperties = $.outputs.prepareTrackPayload.getTrimmedEvent === 'order_completed' ? $.outputs.prepareTrackPayload.updateOrderCompletedEventProperties.properties : $.outputs.prepareTrackPayload.populateEventProperties.properties;
+          {
+          ...$.outputs.prepareTrackPayload.populateEventProperties,
+            "properties": {
+              ...eventProperties,
+              cart: .message.properties@prop.products.({
+                "productId": .sku ?? prop.sku,
+                "quantity": Number(.quantity ?? prop.quantity),
+                "itemPrice": Number(parseFloat(.price ?? prop.price).toFixed(2))
+              })
+            }
+          }
+
+  - name: payload
+    condition: $.outputs.messageType === {{$.EventType.IDENTIFY}}
+    template: |
+      const outputPayload = {
+        ...$.outputs.commonFields,
+        "events": $.outputs.prepareIdentifyPayload
+        }
+    else:
+      name: trackPayload
+      template: |
+        const outputPayload = {
+        ...$.outputs.commonFields,
+        "events": [$.outputs.prepareTrackPayload]
+        }
+
+  - name: buildResponseForProcessTransformation
+    description: build response
+    template: |
+      {
+        "body": {
+          "JSON": $.outputs.payload,
+          "JSON_ARRAY": {},
+          "XML": {},
+          "FORM": {}
+        },
+        "version": "1",
+        "type": "REST",
+        "method": "POST",
+        "endpoint": $.ENDPOINT,
+        "headers": {
+          "DY-API-Key": .destination.Config.apiKey,
+          "Content-Type": "application/json"
+        },
+        "params": {},
+        "files": {}
+      }
```

Comment:
Check if we can use the shared defaultRequestConfig() here.

Reply:
Updated